test/ci/vars.yml.sample

```diff
@@ -0,0 +1,39 @@
+---
+vm_prefix: "ci_test"
+
+type: aws
+aws_user: "ec2-user"
+python: "/usr/bin/python"
+aws_key: "libra"
+aws_region: "us-east-1"
+aws_cluster_id: "ci"
```

Comment:
Just fyi, when we actually go to use this in CI, we need to make sure we randomize this parameter.

Reply:
Yep, in CI this would be set by reading `INSTANCE_PREFIX` env var, which is unique for each run